tsl/test/sql/plan_skip_scan_compressed.sql.in

```diff
@@ -0,0 +1,229 @@
+-- This file and its contents are licensed under the Timescale License.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-TIMESCALE for a copy of the license.
+
+\set PREFIX 'EXPLAIN (ANALYZE, VERBOSE, COSTS OFF, SUMMARY OFF, TIMING OFF)'
+
+--DROP TABLE sensor_data;
+--DROP TABLE sensor_data_ht cascade;
+
+CREATE TABLE sensor_data(
+time timestamptz not null,
+sensor_id integer not null,
+cpu double precision null,
+temperature double precision null );
+
+INSERT INTO sensor_data (time, sensor_id, cpu, temperature) VALUES ('2000-01-01 00:00:00-00', 1, 0.5, 80);
```

Comment:
So do we have only one value per `sensor_id`? Even for uncompressed skip scan, this is not enough to test different ordering by time, because with this data only one result is possible. For compressed skip scan, we need even more data -- many compressed batches per `sensor_id` are required.